packages/ui/src/views/labels/labels-list-page.tsx

```diff
@@ -99,22 +95,12 @@ export const LabelsListPage: FC<LabelsListPageProps> = ({
           <Layout.Vertical grow gapY="none">
             <LabelsListView
               {...labelsListViewProps}
-              labels={spaceLabels}
+              useLabelsStore={useLabelsStore}
               labelContext={{ space: space_ref, repo: repo_ref }}
```

Comment:
This prop doesn't seem to be used in the `LabelsListView`, can be removed here and from type. If needed, it can be get from `useLabelsStore`.